docs/orchestration/concepts/flows.md

```diff
@@ -152,6 +152,14 @@ mutation {
 }
 ```
 
+::: warning Dask Work Stealing with Version Locking
+**If you are using Dask**, we recommend turning off [Dask work stealing](https://distributed.dask.org/en/latest/work-stealing.html) if you enable version locking, as in rare occasions work stealing can result in tasks attempting to run twice which breaks the idempotency constraint of version locking.
```

Comment:
We might want to relax the wording here -> version locking will successfully prevent the task from running twice _but_ at this moment it causes a lot of log noise and results in some inefficient re-submissions of the flow run (to be clear, this is something we should improve)